packages/sdk/contractkit/src/wrappers/MultiSig.ts

```diff
@@ -60,6 +60,13 @@ export class MultiSigWrapper extends BaseWrapper<MultiSig> {
     )
   }
 
+  async confirmTransaction(transactionId: number) {
+    return toTransactionObject(
+      this.connection,
+      this.contract.methods.confirmTransaction(transactionId)
+    )
+  }
```

Comment:
Nit: Any reason you didn't use the existing [`submitOrConfirmTransaction`](https://github.com/celo-org/celo-monorepo/blob/95736497e4f2e15f3247357b7776e6ee2440f7c4/packages/sdk/contractkit/src/wrappers/MultiSig.ts#L51-L54) function and instead created a new `confirmTransaction` in ContractKit?

I can see it makes the function call shorter in [`approve.ts`](https://github.com/celo-org/celo-monorepo/blob/1a0a5b2b1d1a8b97f0e36ac7266d071be4b7a0b3/packages/cli/src/commands/multisig/approve.ts#L44), but in exchange adds complexity to ContractKit's API. 

User's now have a choice between `submitOrConfirmTransaction` and `confirmTransaction`.

I'm not strongly for or against either approach (I haven't spent enough time in CK and the CLI), just curious if there was a reason you decided to add a function.

Reply:
`submitOrConfirmTransaction` will search for and approve if present or else submit a tx matching given bytes -- here the whole idea is the tx already exists, we know it's id, and we definitely don't want to ever submit a new one. So more efficient to expose `confirmTransaction` directly.